recipe/deploy/release.php

```diff
@@ -74,53 +72,34 @@
     return $releases;
 });
 
-/**
- * Return release path.
- */
-set('release_path', function () {
-    $releaseExists = run("if [ -h {{deploy_path}}/release ]; then echo 'true'; fi")->toBool();
-    if (!$releaseExists) {
-        throw new \RuntimeException(
-            "Release path does not found.\n" .
-            "Run deploy:release to create new release."
-        );
-    }
-
-    $link = run("readlink {{deploy_path}}/release")->toString();
-    return substr($link, 0, 1) === '/' ? $link : get('deploy_path') . '/' . $link;
```

Comment:
In this case, I wanted to change `release_dir`variable to `current_release` like capistrano, but I wanted to mantain the PR with not too much changes.